apphomepage/templates/apphomepage/about.html

```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE html>
```

Comment:
I have just notice that settings include application called apphome page
Is it really the name of the application?

Will you have more templates? for this application? consider giving different app name where we could add more templates under this one.

Reply:
I also see anothe PR without reviewers 
https://github.com/beyond-io/picATrip/issues/35 it is dealing with the above

Reply:
Do you suggest to change the name of the application in this PR?
Or create a new PR that will deal with #35 ?

Reply:
@Yarboa okay now? (if yes please resolve)